### ISSUE TITLE
FSNotes Print Margins

### ISSUE THREAD

**Issue by peavine** [glushchenko/fsnotes#227]
I was pleasantly surprised by the new print feature in 1.6.2. It's a convenience to use FSNotes rather than TextEdit to print a note.

I have encountered an issue where the last 3 or 4 characters of every full-length line of text are missing when printed. I deleted the FSNotes container files, but that didn't make a difference.

This appears not to be an issue with the default note font, but is always an issue with my preferred font, which is Courier 14 point. I'll try some other fonts to see if I can find one that prints properly and I find readable.

**Comment by peavine** [comment]
I've done a bit more research on this and have concluded that I'm sticking with Courier 14 point as my note font. I like it and it's easy to read with my poor eyesight.

I created a test note in FSNotes and then printed it to a PDF from within FSNotes and then printed the same file with TextEdit. I also printed this document to my Brother laser printer from within FSNotes and TextEdit and received the same results.

I've attached both PDF files and the differences are:

* TextEdit uses the Courier font to print while FSNotes uses an Arial-like font. 

* TextEdit prints the entire line of text while FSNotes cuts off the last few characters.

Is it possible to have FSNotes print like TextEdit?

Just as an aside, I did look at available settings in the printer dialog (including scale to fit) but none made a difference.

Thanks.

[FSNotes.pdf](https://github.com/glushchenko/fsnotes/files/1971388/FSNotes.pdf)
[TextEdit.pdf](https://github.com/glushchenko/fsnotes/files/1971389/TextEdit.pdf)
